shell/platform/common/client_wrapper/include/flutter/encodable_value.h

```diff
@@ -215,6 +215,11 @@ class EncodableValue : public internal::EncodableValueVariant {
     }
     return std::get<int64_t>(*this);
   }
+
+  // Avoid operator<=> problems with std::variant in C++20 mode.
```

Comment:
What are the problems? Even reading the internal change I'm not clear on what problem is being solved.

And is this permanent, or a temporary workaround? If the latter, what are the conditions for removing it?